src/NServiceBus.SagaAudit.AcceptanceTests/When_a_message_is_handled_by_a_saga.cs

```diff
@@ -0,0 +1,171 @@
+﻿namespace ServiceControl.Plugin.Nsb6.SagaAudit.AcceptanceTests
```

Comment:
This is nit-picky, but should we kill the `Nsb6` portion of the namespace wherever we are using it?